test/e2e/pattern.test.js

```diff
@@ -92,5 +96,25 @@ describe('vl-pattern', async () => {
     await assert.eventually.isFalse(input.hasPricePattern());
     await assert.eventually.isTrue(input.hasRRNPattern());
     await assert.eventually.isFalse(input.hasPatternPrefix());
+    await assert.eventually.isFalse(input.hasUuidPattern());
+  });
+
+  it('als gebruiker kan ik alleen een toegelaten uuid patroon invullen', async () => {
+    const input = await vlPatternPage.getUuidInput();
+    await input.setValue('---XYZ___');
+    await assert.eventually.equal(input.getValue(), '');
+    await input.setValue('ZZZZZZZZ-ZZZZ-ZZZZ-ZZZZ-ZZZZZZZZZZZZ');
+    await assert.eventually.equal(input.getValue(), '');
+    await input.setValue('1c6fa548-5eef-11eb-ae93-0242ac130002');
+    await assert.eventually.equal(input.getValue(), '1c6fa548-5eef-11eb-ae93-0242ac130002');
+    await input.setValue('1c6fa5485eef11ebae930242ac130002');
+    await assert.eventually.equal(input.getValue(), '1c6fa548-5eef-11eb-ae93-0242ac130002');
+    await assert.eventually.isFalse(input.hasIBANPattern());
+    await assert.eventually.isFalse(input.hasPhonePattern());
+    await assert.eventually.isFalse(input.hasDatePattern());
+    await assert.eventually.isFalse(input.hasPricePattern());
+    await assert.eventually.isFalse(input.hasRRNPattern());
+    await assert.eventually.isFalse(input.hasPatternPrefix());
```

Comment:
lijkt mij niet zo nuttig om in elke test opnieuw te checken dat het alle andere patronen NIET is?